metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/CounterWrapper.java

```diff
@@ -32,21 +31,23 @@
  * Servo counter wrapper.
  *
  * @author amajumdar
+ * @since 1.0.0
  */
 @Slf4j
 public final class CounterWrapper {
 
-    private static final LoadingCache<String, Counter> COUNTERS = CacheBuilder.newBuilder()
-        .build(
+    private static final LoadingCache<String, Counter> COUNTER_LOADING_CACHE
```

Comment:
We shouldn't need this class anymore. The Registry already maintains the cache for us.